diff --git a/config.h b/config.h
index 03bfd37..7bf1af4 100644
--- a/config.h
+++ b/config.h
@@ -8,8 +8,8 @@ static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10", "Font Awesome 5 Free:pixelsize=10:antialias=true:autohint=true" };
-static const char dmenufont[]       = "monospace:size=10";
+static const char *fonts[]          = { "monospace:size=@fontSize@", "Font Awesome 5 Free:pixelsize=@fontSize@:antialias=true:autohint=true" };
+static const char dmenufont[]       = "monospace:size=@fontSize@";
 static const char col_gray1[]       = "#222222";
 static const char col_gray2[]       = "#444444";
 static const char col_gray3[]       = "#bbbbbb";
